app/Http/Controllers/DashboardController.php

```diff
@@ -28,6 +29,16 @@ public function __construct()
      */
     public function index(Request $request)
     {
+        if (Gate::denies('index-dashboard'))
+        {
+            if (\Request::ajax()) {
+                return response()->json("Please contact the website's admin to request access.", 403);
+            } else {
+                return redirect()->route('home')
+                    ->with('failure', 'Please contact the website\'s admin to request access.');
```

Comment:
Message should contain the email address engr-smartsettia@uidaho.edu.

```Please contact engr-smartsettia@uidaho.edu to request access.```

Reply:
Updated in commit f99838b